### DISCUSSION TITLE
The latest "pypi situation"

### DISCUSSION THREAD

**Discussion by jackiekazil** [Open-Source-Software-Neighborhood-Watch/town-square#17] [General]
https://www.b-list.org/weblog/2022/jul/11/pypi/ 

^^ The author James Bennet does a lot of things in the Django community - https://www.b-list.org/projects/ 

tldr;

* pypi is rolling out required 2FA for "critical" packages 
* pypi is making maintainers sign up
* people have opinions

Wondering if we have opinions.

**Comment by jspeed-meyers** [comment]
I'm relatively supportive, but not a zealot about this. wdyt, @jackiekazil? Sorry I didn't see this thread until now.

**Reply by jackiekazil** [reply]
No worries and same. There is a decent amount of content between the multiple posts to come to the same conclusion, which is meh. But feels like I should have a stronger opinion... but it make sense & I can see why they choose the approach they did.
